src/main/java/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,14 @@
+package bookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Getter;
+
+@Getter
```

Comment:
```suggestion
@Data
```

Reply:
the previous spring topic said that we shouldn't use @Data annotation because it's insecure for toString, equals, and hashcode methods